recipes/neomodel/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "neomodel" %}
+{% set version = "3.2.5" %}
+{% set sha256 = "3138e960b0524afed6dc212513636e1b11103a69ce598757111763d17cef0a43" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - neo4j-python-driver
```

Comment:
It looks like `neo4j-python-driver` and `pytz` are not required to build the package.